build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
     ext {
         gradleVersion = '7.3.3'
         assertJVersion = '3.24.2'
-        atdVersion = '5b23b2273f'
+        atdVersion = '13.1.0'
```

Comment:
This is the correct ATD version - 5b23b2273f

Reply:
Updated to atdVersion to 5b23b2273f